data/pages-content/volunteers.md

```diff
@@ -20,7 +20,7 @@ In return, we will have a small number of perks for you, some small gifts to say
 # How to sign up
   - Fill this form and we will contact and onboard you. https://forms.gle/Y472nH5m8ZRWpbVP8
   - If you want to save some time you can:
-      + Acknowledge our [EuroPython 2022 Data Privacy Policy](https://docs.google.com/forms/d/e/1FAIpQLSdNTLsFn-YKD4l6icY4NEhmVG5JRyawm-QcvZUQ9HSylEXldw/viewform). This is needed, since many of you will have to handle private data as part of e.g. helping at the registration desk.
+      + Acknowledge our [EuroPython 2023Data Privacy Policy](https://docs.google.com/forms/d/e/1FAIpQLSdNTLsFn-YKD4l6icY4NEhmVG5JRyawm-QcvZUQ9HSylEXldw/viewform). This is needed, since many of you will have to handle private data as part of e.g. helping at the registration desk.
```

Comment:
```suggestion
      + Acknowledge our [EuroPython 2023 Data Privacy Policy](https://docs.google.com/forms/d/e/1FAIpQLSdNTLsFn-YKD4l6icY4NEhmVG5JRyawm-QcvZUQ9HSylEXldw/viewform). This is needed, since many of you will have to handle private data as part of e.g. helping at the registration desk.
```